src/components/People/PeoplePage.tsx

```diff
@@ -0,0 +1,57 @@
+import { PeopleFilters } from '../PeopleFilters';
+import { Loader } from '../Loader';
+import { PeopleTable } from './PeopleTable';
+import { Person } from '../../types';
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [isLoaded, setIsloaded] = useState<boolean>(true);
```

Comment:
this name for state sounds like if data has arrived. `isLoading` would be better